.kitchen.yml

```diff
@@ -10,7 +10,9 @@ provisioner:
   data_bags_path: test/data_bags
 
 platforms:
-  - name: ubuntu-16.04
+  # Ubuntu 18.04 is what’s used on the live OSM servers,
+  # according to https://hardware.openstreetmap.org/
+  - name: ubuntu-18.04
```

Comment:
I updated this on master last week, so it shouldn't be needed here if you rebase.

Reply:
Merged in upstream, thank you!